### PR TITLE
Delegated webhooks

### PR DESCRIPTION
**Pull request by nkramer**

### CODE REVIEW

File: api-reference/beta/api/subscription-post-subscriptions.md [hunk 1]
```diff
@@ -41,7 +41,11 @@ Creating a subscription requires read permission to the resource. For example, t
 
 ### chatMessage (Microsoft Teams)
 
-**chatMessage** subscriptions require [encryption](/graph/webhooks-with-resource-data). Subscription creation will fail if [encryptionCertificate](../resources/subscription.md) is not specified. Before creating a **chatMessage** subscription, you must request access. For details, see [Protected APIs in Microsoft Teams](/graph/teams-protected-apis). 
+**chatMessage** subscriptions with delegated permissions do not support resource data (includeResourceData must be false), 
+and do not require [encryption](/graph/webhooks-with-resource-data).
+
+**chatMessage** subscriptions with application permissions include resource data, and require [encryption](/graph/webhooks-with-resource-data). Subscription creation will fail if [encryptionCertificate](../resources/subscription.md) is not specified. Before creating a **chatMessage** subscription, you must request access. For details, see [Protected APIs in Microsoft Teams](/graph/teams-protected-apis).
+Delegated permissions
```
Comment (baywet) [inline api-reference/beta/api/subscription-post-subscriptions.md:48]:
```
```

Comment (baywet) [reply api-reference/beta/api/subscription-post-subscriptions.md:48]:
```suggestion
```

Comment (nkramer) [reply api-reference/beta/api/subscription-post-subscriptions.md]: fixed

File: api-reference/beta/api/subscription-post-subscriptions.md [hunk 2]
```diff
@@ -41,7 +41,10 @@ Creating a subscription requires read permission to the resource. For example, t
 
 ### chatMessage (Microsoft Teams)
 
-**chatMessage** subscriptions require [encryption](/graph/webhooks-with-resource-data). Subscription creation will fail if [encryptionCertificate](../resources/subscription.md) is not specified. Before creating a **chatMessage** subscription, you must request access. For details, see [Protected APIs in Microsoft Teams](/graph/teams-protected-apis). 
+**chatMessage** subscriptions with delegated permissions do not support resource data (includeResourceData must be false), 
+and do not require [encryption](/graph/webhooks-with-resource-data).
+
+**chatMessage** subscriptions with application permissions include resource data, and require [encryption](/graph/webhooks-with-resource-data). Subscription creation will fail if [encryptionCertificate](../resources/subscription.md) is not specified. Before creating a **chatMessage** subscription, you must request access. For details, see [Protected APIs in Microsoft Teams](/graph/teams-protected-apis).
```
Comment (RamjotSingh) [inline api-reference/beta/api/subscription-post-subscriptions.md:47]: This is not true. Subscriptions require no approval.

Comment (baywet) [reply api-reference/beta/api/subscription-post-subscriptions.md:47]: @RamjotSingh is this something that's changed in the last few months? I remember having to go through approval myself a few months ago when I was testing the "all" endpoints.

Comment (nkramer) [reply api-reference/beta/api/subscription-post-subscriptions.md:47]: This is orthogonal to the feature we're introducing here -- we need to close on what we're doing for protected APIs for application permissions webhooks.

**Pull request merged**